src/seedu/addressbook/AddressBook.java

```diff
@@ -278,15 +278,18 @@ private static void processProgramArgs(String[] args) {
      */
     private static void setupGivenFileForStorage(String filePath) {
 
-        if (!isValidFilePath(filePath)) {
-            showToUser(String.format(MESSAGE_INVALID_FILE, filePath));
-            exitProgram();
-        }
-
+        checkValidFilePath(filePath);
         storageFilePath = filePath;
         createFileIfMissing(filePath);
     }
 
+	private static void checkValidFilePath(String filePath) {
```

Comment:
as mentioned in class, using `tabs` instead of four `spaces` for indentation is coding standard violation.